src/test/java/com/ericsson/ei/subscriptionhandler/test/SubscriptionHandlerTest.java

```diff
@@ -288,18 +308,29 @@ public void sendMailTest() {
 
     @Test
     public void testRestPostTrigger() throws IOException {
-        when(springRestTemplate.postDataMultiValue(url, mapNotificationMessage(), headerContentMediaType))
+        when(springRestTemplate.postDataMultiValue(url, mapNotificationMessage(subscriptionData), headerContentMediaType))
                 .thenReturn(STATUS_OK);
         subscription.informSubscriber(aggregatedObject, new ObjectMapper().readTree(subscriptionData));
-        verify(springRestTemplate, times(1)).postDataMultiValue(url, mapNotificationMessage(), headerContentMediaType);
+        verify(springRestTemplate, times(1)).postDataMultiValue(url, mapNotificationMessage(subscriptionData), headerContentMediaType);
+    }
+    
+    @Test
+    public void testRestPostTriggerForJenkins() throws IOException {
+        String formkey = "Authorization";
```

Comment:
private static final and up in the class

Reply:
Done